apps/chat/src/hooks/useTokenizer.ts

```diff
@@ -2,23 +2,17 @@ import { useCallback, useEffect, useState } from 'react';
 
 import { DialAIEntityModel } from '../types/models';
 
-import { Tiktoken, get_encoding } from '@dqbd/tiktoken';
+import { Tiktoken, getEncoding } from 'js-tiktoken';
```

Comment:
@dqbd/tiktoken
we still have places which use this lib. Do we need them both: `js-tiktoken` and `@dqbd/tiktoken`. Can we get rid of `@dqbd/tiktoken`?